components/dropdown/README.md

```diff
@@ -85,3 +85,25 @@ Opt-in prop to show popovers fullscreen on mobile, pass `false` in this prop to
  - Type: `Boolean`
  - Required: No
  - Default: `false`
+
+## draggable
+
+A flag to specify whether element should be draggable, as per the HTML5 Drag & Drop API.
+
+ - Type: `Boolean`
+ - Required: No
+ - Default: `false`
+
+## onDragStart
+
+A callback invoked when the element is first dragged - an event listener as per the HTML5 Drag & Drop API.
+
+- Type: `Function`
+- Required: No
+
+## onDragEnd
+
+A callback invoked when a drag operation is over - an event listener as per the HTML5 Drag & Drop API.
+
+- Type: `Function`
+- Required: No
```

Comment:
Why do we have to add these props to the `DropDown` component. This component seems unrelated to drag and drop?

Reply:
Actually, it is necessary. The dropdown component covers a tiny bit of area around the ellipsis, which we want to make draggable. Without it you cannot grab the block from around the ellipsis on the right. Basically, any element that we want to be able to grab the block from needs to apply the `onDragStart` and `onDragEnd` props (passed from the block).

Reply:
Good point @aduth . I will try and work this out in a different way. :-)

Reply:
@aduth I reworked this to not pass the handlers to these UI components. Please review my last couple of commits to see if this is any better. :-)